contracts/augur/reporting/IMarket.sol

```diff
@@ -1,50 +1,14 @@
 pragma solidity 0.5.10;
 
 
-import 'ROOT/libraries/IOwnable.sol';
-import 'ROOT/ICash.sol';
-import 'ROOT/reporting/IUniverse.sol';
-import 'ROOT/reporting/IDisputeWindow.sol';
-import 'ROOT/reporting/IShareToken.sol';
-import 'ROOT/reporting/IReportingParticipant.sol';
-import 'ROOT/reporting/IV2ReputationToken.sol';
-import 'ROOT/reporting/IInitialReporter.sol';
-import 'ROOT/IAugur.sol';
-
-
-contract IMarket is IOwnable {
+contract IMarket {
     enum MarketType {
         YES_NO,
         CATEGORICAL,
         SCALAR
     }
 
-    function initialize(IAugur _augur, IUniverse _universe, uint256 _endTime, uint256 _feePerCashInAttoCash, uint256 _affiliateFeeDivisor, address _designatedReporterAddress, address _creator, uint256 _numOutcomes, uint256 _numTicks) public;
-    function derivePayoutDistributionHash(uint256[] memory _payoutNumerators) public view returns (bytes32);
-    function doInitialReport(uint256[] memory _payoutNumerators, string memory _description, uint256 _additionalStake) public returns (bool);
-    function getUniverse() public view returns (IUniverse);
-    function getDisputeWindow() public view returns (IDisputeWindow);
     function getNumberOfOutcomes() public view returns (uint256);
     function getNumTicks() public view returns (uint256);
-    function getMarketCreatorSettlementFeeDivisor() public view returns (uint256);
-    function getForkingMarket() public view returns (IMarket _market);
-    function getEndTime() public view returns (uint256);
-    function getWinningPayoutDistributionHash() public view returns (bytes32);
-    function getWinningPayoutNumerator(uint256 _outcome) public view returns (uint256);
-    function getWinningReportingParticipant() public view returns (IReportingParticipant);
```

Comment:
These are needed (or some equivalent is needed) to determine payouts for claiming winnings